wasmbinding/queries.go

```diff
@@ -226,6 +260,16 @@ func dexQuery[T, R any](ctx sdk.Context, query *T, queryHandler func(ctx context
 	return data, nil
 }
 
+func processResponse(resp interface{}, err error) ([]byte, error) {
```

Comment:
hm, seems like if you rename `dexQuery` to something more generic like `handleQuery` you could reuse it. The functions are mostly the same.

Reply:
i personally don't want to refactor dex queries in this pr, but i like overall idea. let's create a task to refactor this?

Reply:
Lets create a task to refactor all bindings handlers